src/handlers/swap_handlers.ts

```diff
@@ -183,6 +183,10 @@ export class SwapHandlers {
                 req.query.excludedSources === undefined
                     ? []
                     : parseUtils.parseStringArrForERC20BridgeSources((req.query.excludedSources as string).split(',')),
+            includedSources:
+                req.query.includedSources === undefined
+                    ? []
+                    : parseUtils.parseStringArrForERC20BridgeSources((req.query.includedSources as string).split(',')),
```

Comment:
`includedSources` can only be `RFQT`. Doesn't this added code mean that it could be any ERC20 source?

Reply:
Yes. We now support an `includedSources` option in AS.

Reply:
This change should still allow you to ask for RFQT only.